docs/suggest-sig.md

```diff
@@ -0,0 +1,166 @@
+# Suggesting Sigs
+
+It's possible to build Sorbet in a way that it can suggest the "most impactful"
+methods to add a signature to, something like this:
+
+```
+❯ sorbet --suggest-sig foo.rb
+No errors! Great job.
+Typing `::A#bar` would impact 100.0% callsites(2 out of 2).
+```
+
+There are some fairly involved setup steps to get this working.
+
+Something less involved is asking Sorbet to directly edit a project's source
+code with sigs that might work. This involves involves less custom setup.
```

Comment:
nit: involves is repeated twice

Reply:
https://github.com/sorbet/sorbet/pull/2903